docs/content/preview/explore/query-1-performance/yb-pg-stat-get-queries.md

```diff
@@ -0,0 +1,272 @@
+---
+title: View terminated queries with yb_pg_stat_get_queries
+linkTitle: View terminated queries
+description: Track planning and execution statistics for all SQL statements executed by a server.
+headerTitle: View terminated queries with yb_pg_stat_get_queries
+headcontent: See why a query failed
+menu:
+  preview:
+    identifier: yb-pg-stat-get-queries
+    parent: query-tuning
+    weight: 350
+type: docs
+---
+
+Use the YugabyteDB `yb_pg_stat_get_queries` function to see terminated queries and the reason for their termination.
+
+When a query quits for unexpected reasons, information about the query information and the responsible backend is stored and you can access it using yb_pg_stat_get_queries. Calling the module returns queries using the following criteria:
```

Comment:
Suggestion:

When a query quits for unexpected reasons, information about the query information and the responsible backend is stored. You can access this information by using `yb_pg_stat_get_queries`.